Functions/Assertions/Should.ps1

```diff
@@ -83,11 +84,13 @@ function Should {
             $testFailed = Get-TestResult $parsedArgs $value
 
             if ($testFailed) {
+                $ShouldExceptionLineText = $MyInvocation.Line.TrimEnd("`n")
```

Comment:
Works for me.  In the short term, I plan to refactor any tests that have long, multi-line Should assertions, and assign the result to a descriptive variable name.  That way my output will always be tidy:  `##:   $somethingMeaninful | Should Be $somethingCorrect`